src/styles/blocks/article.scss

```
@@ -0,0 +1,59 @@
+.article {
+
+  display: flex;
```

```suggestion
.article {
  display: flex;
```
remove all those unnecessary white lines